zkevm-circuits/src/evm_circuit/util.rs

```diff
@@ -250,15 +253,15 @@ pub(crate) fn evm_cm_distribute_advice<F: Field>(
     // Mark columns used for byte lookup
     #[allow(clippy::reversed_empty_ranges)]
     for _ in 0..N_U8_LOOKUPS {
-        dist.add(CellType::LookupU8, advices[column_idx]);
+        dist.add(CellType::Lookup(Table::U8), advices[column_idx]);
         assert_eq!(advices[column_idx].column_type().phase(), 0);
         column_idx += 1;
     }
 
     // Mark columns used for byte lookup
     #[allow(clippy::reversed_empty_ranges)]
     for _ in 0..N_U16_LOOKUPS {
-        dist.add(CellType::LookupU16, advices[column_idx]);
+        dist.add(CellType::Lookup(Table::U8), advices[column_idx]);
```

Comment:
Here got typo.
Test still passed, suggest adding some unittest to check that

Reply:
Oh dang, I expected this mistake so I reviewed it twice locally before committing still messed it up :( Thanks for pointing!